packages/transition-frontend/webpack.config.js

```diff
@@ -18,6 +18,9 @@ if (!process.env.NODE_ENV) {
     process.env.NODE_ENV = 'development';
 }
 
+// Make sure Transition specific config are loaded
+require('transition-common/lib/config/project.config');
```

Comment:
isn't a bad idea to expose the project config directly into the frontend???

Reply:
> isn't a bad idea to expose the project config directly into the frontend???

yeah, and we've known it for a while now: #424 At least we [try to] keep in the backend the more sensible stuff.

Reply:
Should we try to not add more then? :)